src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  line-height: 60px;
+  background: white;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+  text-decoration: none;
+}
```

Comment:
Be consistent with your vertical margins. You've added a left margin to '.nav__item', but vertical margins should only be top or bottom, not both or left/right.